### DISCUSSION TITLE
Is there a way to programatically activate the Tab?

### DISCUSSION THREAD

**Discussion by alex-popov-tech** [wezterm/wezterm#3159] [Q&A] [answered]
Hello, I am migrating from tmux, and wanted to have preset of tabs/panes when i open wezterm.

A bit of problem I have now, is that if i spawn 7 tabs, last tab is always active, and i want first tab to be active in the end.

Is there an ability to activate tab ( I saw it is in actions, maybe i can use it ? ) or spawn new tab without making it active?

**Comment by wez** [comment, marked as answer]
From what context are you spawning / hoping to control this?

**Reply by alex-popov-tech** [reply]
from gui-startup event

**Reply by wez** [reply]
https://wezfurlong.org/wezterm/config/lua/mux-window/gui_window.html should let you get a gui window from a mux window.
From there, https://wezfurlong.org/wezterm/config/lua/window/perform_action.html will let you evaluate https://wezfurlong.org/wezterm/config/lua/keyassignment/ActivateTab.html

**Reply by alex-popov-tech** [reply]
that works, thank you!
